app/assets/stylesheets/components/_accordions.scss

```diff
@@ -1,35 +1,51 @@
 .faq-accordion-button,
-.moving-guide-accordion-button {
+.moving-guide-accordion-button,
+.moving-tips-accordion-button {
   background-image: asset-url('minus-blue.png');
   background-image: asset-url('minus-blue.svg');
 }
 
 .faq-accordion-button[aria-expanded=false],
-.moving-guide-accordion-button[aria-expanded=false] {
+.moving-guide-accordion-button[aria-expanded=false],
+.moving-tips-accordion-button[aria-expanded=false] {
   background-image: asset-url('plus-blue.png');
   background-image: asset-url('plus-blue.svg');
 }
 
-.moving-guide-accordion-button {
+.moving-guide-accordion-button,
+.moving-tips-accordion-button {
   background-position: right 1.5rem center;
   padding: 1.5rem 3.5rem 1.5rem 1.5rem;
 
   div {
     display: table;
   }
 
-  img {
-    height: 42px;
-    width: 42px;
+  span {
+    display: table-cell;
+    vertical-align: middle;
   }
 
-  span {
-    padding-left: 1.5rem;
+  img {
+    margin-right: 1.5rem;
   }
+}
 
-  img,
-  span {
+.moving-guide-accordion-button img {
+  display: table-cell;
+  height: 42px;
+  vertical-align: middle;
+  width: 42px;
+}
+
+.moving-tips-accordion-button img {
+  display: none;
+
+  @include media($medium-screen) {
     display: table-cell;
+    height: 50px;
+    max-width: 55px;
+    width: 55px;
     vertical-align: middle;
   }
 }
```

Comment:
Would it be possible to consolidate the styling (and class name) of these two accordions? Looking at the code here, there's a lot of overlap.

Reply:
The main difference is that the moving tips one has super long accordion button text, so for a narrow screen the icon disappears. The moving guide accordion only appears for narrow screens (for wider screens it's a table).

Reply:
Ah, gotcha. Thanks for clarifying!